src/Compute/Compute/ChangeLog.md

```diff
@@ -22,6 +22,9 @@
 ## Upcoming Release
 * Updated `New-AzVM`, `New-AzVmss`, `Update-AzVM`, and `Update-AzVmss` to pass `Standard` as an input of `-SecurityType` parameter.
 * Added breaking change message for `Get-AzVMSize`.
+* Added new parameters `ZonePlacementPolicy`, `IncludeZone`, `ExcludeZone`, and `AlignRegionalDisksToVMZone` to cmdlets `New-AzVM` and `New-AzVmConfig`
```

Comment:
Please add new changelogs right after **Upcoming Release** to avoid potential issues when we merge release branch back to main branch.

Reply:
done as requested